### PR TITLE
Core: Feature - Expose sync methods

### PR DESCRIPTION
**Pull request by cjmconie**

### CODE REVIEW

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 1]
```diff
@@ -105,6 +111,9 @@ extension BLOCKv {
     /// `VatomModel`. The array contains *packaged* vAtoms. Packaged vAtoms have their template's configured Faces
     /// and Actions as properties.
     ///
+    /// ## Asynchronous Behavior
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift:114]: isn't this missing a `u`

File: Sources/Core/Requests/BLOCKv+SyncRequests.swift [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+//
+
+import Foundation
+
+/// This extension groups together all BLOCKv vatom requests.
+extension BLOCKv {
+    
+    /// Fetches the current user's inventory hash.
+    ///
+    /// - Parameter completion: Completion hanlder that so called once the request completes.
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+SyncRequests.swift]: `handler ?`

File: Sources/Core/Requests/BLOCKv+SyncRequests.swift [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        }
+        
+    }
+    
+    /// Fetches the current user’s inventory vatoms’ sync numbers.
+    ///
+    /// - Parameters:
+    ///   - limit: Paging limit.
+    ///   - token: Paging token.
+    ///   - completion: Completion hanlder that so called once the request completes.
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+SyncRequests.swift]: `handler`

File: Sources/Core/Requests/BLOCKv+SyncRequests.swift [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        }
+        
+    }
+    
+    /// Fetches the face changes for the specified template ids after the specified time.
+    ///
+    /// - Parameters:
+    ///   - templateIds: List of local template ids whose changes should be fetched.
+    ///   - since: Unix epoch after which changes should be fetched (measured in milliseconds).
+    ///   - completion: Completion hanlder that so called once the request completes.
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+SyncRequests.swift]: `handler`

File: Sources/Core/Requests/BLOCKv+SyncRequests.swift [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        }
+        
+    }
+    
+    /// Fetches the action changes for the specified template ids after the specified time.
+    ///
+    /// - Parameters:
+    ///   - templateIds: List of local template ids whose changes should be fetched.
+    ///   - since: Unix epoch after which changes should be fetched (measured in milliseconds).
+    ///   - completion: Completion hanlder that so called once the request completes.
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+SyncRequests.swift]: `Handler`

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 6]
```diff
@@ -20,6 +20,9 @@ extension BLOCKv {
...
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift:23]: `Behaviour`

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 7]
```diff
@@ -20,6 +20,9 @@ extension BLOCKv {
...
+    /// This method is **non-deterministc**.
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift]: `deterministic`

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 8]
```diff
@@ -64,6 +67,9 @@ extension BLOCKv {
     /// Fetches a vatom by its identifier. The completion handler passes in a `VatomModel` that has been *packaged*.
     /// Packaged vAtoms have their template's configured Faces and Actions as properties.
     ///
+    /// ## Asynchronous Behavior
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift:70]: `Behaviour`

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 9]
```diff
@@ -64,6 +67,9 @@ extension BLOCKv {
...
+    /// This method is **deterministc**.
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift]: deterministic

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 10]
```diff
@@ -105,6 +111,9 @@ extension BLOCKv {
     /// `VatomModel`. The array contains *packaged* vAtoms. Packaged vAtoms have their template's configured Faces
     /// and Actions as properties.
     ///
+    /// ## Asynchronous Behavior
+    /// This method is **deterministic** when sipplied with a single vatom id, and  **non-deterministc** for two or more vatom ids.
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift]: supplied

Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift]: deterministic

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 11]
```diff
@@ -142,6 +151,9 @@ extension BLOCKv {
     ///
     /// This will remove the vAtom from the current user's inventory.
     ///
+    /// ## Asynchronous Behavior
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift:154]: behaviour and deterministic

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 12]
```diff
@@ -171,6 +183,9 @@ extension BLOCKv {
 
     /// Sets the parent ID of the specified vatom.
     ///
+    /// ## Asynchronous Behavior
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift:186]: behaviour and deterministic

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift [hunk 13]
```diff
@@ -232,6 +247,9 @@ extension BLOCKv {
 
     /// Searches for vAtoms on the BLOCKv platform.
     ///
+    /// ## Asynchronous Behavior
```
Comment (MalcolmnDEV) [inline Sources/Core/Requests/BLOCKv+VatomRequest.swift:250]: behaviour and deterministic

**Review by MalcolmnDEV — CHANGES_REQUESTED** [review]

File: Sources/Core/Requests/BLOCKv+VatomRequest.swift (hunk 1 above)
Comment (cjmconie) [reply Sources/Core/Requests/BLOCKv+VatomRequest.swift:114]: Behavior is the American English spelling which the repo uses.

**Review by MalcolmnDEV — APPROVED** [review]

**Pull request merged**